site/docs/characteristics/accordionView.mdx

```diff
@@ -0,0 +1,8 @@
+---
+title: Characteristic types
+layout: DetailTechnical
+sidebar:
+  label: Characteristic types
+---
+
+<CssAccordionView />
```

Comment:
We already have a site `Accordion` component, could we use that instead?

Reply:
Also it would be good to the page's content in mdx as much as possible so anyone can edit it

Reply:
Most of the content in rendered dynamically vs the static view on storybook, thus why everything is done inside components rather than mdx. So any changes in css vars over time will automatically get updated rather than manually editting it.